linera-indexer/src/operations.rs

```diff
@@ -131,20 +141,10 @@ where
     }
 }
 
-fn hashed_operation(operation: ChainOperation<bool>) -> ChainOperation<CryptoHash> {
-    let hash = CryptoHash::new(&operation.operation);
-    ChainOperation {
-        operation: operation.operation,
-        height: operation.height,
-        previous_operation: operation.previous_operation,
-        operation_index: operation.operation_index,
-        hash,
-    }
-}
-
 #[derive(OneofObject)]
 pub enum OperationKeyKind {
     Hash(CryptoHash),
```

Comment:
Do we need operations by hash at all?

Reply:
might be practical to refer to an operation with an ID